### PR TITLE
fix(indexer): don't error on empty blocks_synced table

### PR DESCRIPTION
**Pull request by frrist**

### CODE REVIEW

File: services/indexer/indexer.go [hunk 1]
```diff
@@ -195,9 +195,9 @@ func (i *Indexer) mostRecentlySyncedBlockHeight(ctx context.Context) (cid.Cid, i
 	task, err := i.storage.MostRecentSyncedBlock(ctx)
 	if err != nil {
 		if err == pg.ErrNoRows {
-			return i.genesis.Cids()[0], 0, xerrors.Errorf("query recent synced: %w", err)
+			return i.genesis.Cids()[0], 0, nil
 		}
-		return cid.Undef, 0, err
+		return cid.Undef, 0, xerrors.Errorf("query recent synced: %w", err)
```
Comment (placer14) [inline services/indexer/indexer.go:200]: I think the first return value is wrong.

**Review by placer14 — CHANGES_REQUESTED** [review]
Check the first arg.

File: services/indexer/indexer.go (hunk 1 above)
Comment (frrist) [reply services/indexer/indexer.go:200]: Can you explain more?

Comment (placer14) [reply services/indexer/indexer.go:200]: Ah, I misunderstood what was happening here in my haste to review. GTM

**Review by placer14 — APPROVED** [review]
False alarm. LGTM

**Pull request merged**